docs/getting_started.rst

```diff
@@ -26,13 +42,17 @@ In ``fast_pauli``, we represent pauli matrices using the ``Pauli`` class. For ex
     pauli_y = fp.Pauli('Y')
     pauli_z = fp.Pauli('Z')
 
-We can also multiply two ``Pauli`` objects together to get a ``Pauli`` object representing the tensor product of the two pauli matrices.
+    str(pauli_0)  # returns "I"
+
+We can also multiply two ``Pauli`` objects together to get a ``Pauli`` object representing the matrix product of the two pauli matrices.
+The result includes a phase factor because the product of two Pauli matrices is not another Pauli matrix.
```

Comment:
```suggestion
The result includes a phase factor because the product of two Pauli matrices is not another Pauli matrix, but instead a Pauli matrix times by either `i` or `-i` which here we call the phase factor.
```